scripts/install-windows-prereqs.ps1

```diff
@@ -37,18 +37,6 @@ Param(
     [Parameter(mandatory=$true)][ValidateSet("None", "Azure")][string]$DCAPClientType
 )
 
-if ($LaunchConfiguration -eq "SGX1")
-{
-    Write-Host "**** Installing PSW 2.2 ****"
-
```

Comment:
We would need to update https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/Contributors/WindowsManualSGX1Prereqs.md as well.

Reply:
Besides updating from PSW 2.2 to 2.4, I need to figure out how to install sgx_enclave_common.dll into windows/system32 manually. @johnkord?

Reply:
Added manual instructions for Windows SGX1 machines.